lib/api/core/httpRouter/routeHandler.js

```diff
@@ -48,7 +52,11 @@ class RouteHandler {
         this.data.jwt = headers[k].substring('Bearer '.length);
       }
       else if (k.toLowerCase() === 'x-kuzzle-volatile') {
-        this.data.volatile = JSON.parse(headers[k]);
+        try {
+          this.data.volatile = JSON.parse(headers[k]);
+        } catch (e) {
+          throw new BadRequestError('Unable to convert HTTP x-kuzzle-volatile header to JSON')
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)